api/services/Flairs.js

```diff
@@ -3,6 +3,8 @@ var _ = require('lodash');
 var referenceService = require('./References.js');
 var NodeCache = require('node-cache');
 var app_claim_cache = new NodeCache({stdTTL: 300});
+var extraFlair = require('../../config/local.js').extraFlair;
```

Comment:
I would recommend using `sails.config.extraFlair` instead, so that it can be overridden in other config files if necessary.